src/darwin/Framework/CHIP/MTRDeviceController.h

```diff
@@ -177,16 +182,16 @@ typedef void (^MTRDeviceConnectionCallback)(MTRBaseDevice * _Nullable device, NS
 /**
  * Start scanning for commissionable devices.
  *
- * This method will fail if the controller factory is not running.
+ * This method will fail if the controller factory is not running or the browse has already been started.
  */
-- (BOOL)startScan:(id<MTRCommissionableBrowserDelegate>)delegate queue:(dispatch_queue_t)queue MTR_NEWLY_AVAILABLE;
+- (BOOL)startBrowse:(id<MTRCommissionableBrowserDelegate>)delegate queue:(dispatch_queue_t)queue MTR_NEWLY_AVAILABLE;
```

Comment:
Again, I think we want to make the naming clear that we are browsing for commissionables.

Reply:
I changed the name to be `startBrowseForCommissionables`.